src/Adapters/DatabasePendingApprovalRetriever.php

```diff
@@ -40,6 +40,11 @@ public function getPendingApprovalsForApprover( int $approverId ): array {
 	private function queryPendingApprovals( array $categories ): IResultWrapper {
 		$latestApprovalSubquery = $this->getLatestApprovalSubquery();
 
+		$normalizedCategories = array_map(
+			fn( $category ) => str_replace( ' ', '_', $category ),
+			$categories
+		);
```

Comment:
Better to use MediaWiki's normalization?

I'm actually not sure what the best place for such normalization is. Probably presentation layer.